README.md

```diff
@@ -13,10 +13,15 @@ we are going slow because they are not from math backgrou
 - Day6 some live practice
 - Day7 modulo operator & some practice
 - Day8 assignment day
-- Day9 conditional statements, nested if-else, practice
+- Day9 conditional statements,
+Do's and Dont's of Hacktoberfest
+Don't: open pull requests with minor text edits on other people's repositories
+If you're looking for a faster way to open four PR's and win, there's a better way!
+
+Do: Follow a repository's nested if-else, practice
```

Comment:
why you added these do's and dont's in between

Reply:
I think it paste it by mistake i doesn't saw it, sorry for that.

Reply:
ohk